py-polars/Cargo.toml

```diff
@@ -237,6 +237,7 @@ all = [
   "sql",
   "binary_encoding",
   "ffi_plugin",
+  # "new_streaming",
```

Comment:
Was this intended?

Reply:
It wasn't, but I don't think it hurts for easy commenting/uncommenting either.